app/templates/cp_nigeria/steps/scenario_demand.html

```diff
@@ -5,7 +5,7 @@
 
 
 {% block start_body_scripts %}
-<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```

Comment:
latest is not the latest version?

Reply:
for some reason it doesn't seem like it. I found someone else wondering the same thing https://community.plotly.com/t/why-is-plotly-latest-still-1-58-5/59835 but it is unanswered